File: front-end/src/app/service-models/app/v1/dummy-teacher-s-m.ts
import { CoinManagementServiceModelBase } from '../base/coin-management-service-model-base';

export class DummyTeacherSM extends CoinManagementServiceModelBase<number> {
    firstName!: string;
    lastName!: string;
    emailAddress!: string;
    profilePictureFileId?: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/DummyTeacherSM.cs
using Siffrum.Web.Payroll.ServiceModels.Base;

namespace Siffrum.Web.Payroll.ServiceModels.v1
{
    public class DummyTeacherSM : SiffrumPayrollServiceModelBase<int>
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string? EmailAddress { get; set; }
        public int? ProfilePictureFileId { get; set; }
    }
}
